requirements/requirements.in

```diff
@@ -39,6 +39,7 @@ django_mptt
 django_reversion
 django_taggit
 djangorestframework
+drf-extensions
```

Comment:
@jnm, as mentioned in standup, this is due to the `-r requirements.in` present in [`external_services.in`](https://github.com/kobotoolbox/kpi/blob/0b9767fe044edf1dc37eca7c3725dc7ad2675151/requirements/external_services.in#L3)